index.js

```diff
@@ -1,88 +1,119 @@
 // ***** Question 1 *****
+
+let counter = 1;
+counter = 2;
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red"); 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log(counter) ;
+// => 2
+console.log("%c----------", "color: red");
 
 
 // ***** Question 2 *****
+const name = "Gabriel"
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red");
 
-// console.log(name)
-// => "Raffy"
+console.log(name);
+//=> "Raffy"
 // name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+//=> TypeError
+console.log("%c----------", "color: red");
 
 
 
 // ***** Question 3 *****
+function drinkWater(currentThirstLevel){
+  console.log("Man I sure am thirsty");
+  currentThirstLevel-=1; 
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+
+}
```

Comment:
This is great! Two ways to refactor: line 31 can be currentThirstLevel--
Or we can combine lines 31 and 33
return currentThirstLevel - 1